py/py3/ocutils.py

```diff
@@ -7,16 +7,28 @@
 # coding=utf8
 import heapq
 import ocGraph
-import occam
 import re
 import sys
 import time
 
+from enum import Enum
+from vbm_manager import VBMManager
+from sbm_manager import SBMManager
+from report import SeparatorType
+from manager import SearchDirection, SearchType
+
 totalgen = 0
 totalkept = 0
 max_memory_to_use = 8 * 2 ** 30
 
 
+class ActionType(Enum):
```

Comment:
We could also consider throwing all these enums that might be used elsewhere into another file like a `common.py` or something similar

Reply:
Yes, but we'll do that when we write our versions of `fit, sbsearch`. An this particular enum will go away when we do that

Reply:
They should also reside next to classes which they relate to. For example `ModelType` would have to be in `model.py` as it directly relates to it. You don't want to dump all possible enums in a single file and then use them from there